tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreSampleAppsTest.cs

```diff
@@ -275,6 +276,43 @@ public void Builds_NetCore31App_UsingNetCore31_DotNetSdkVersion()
                 result.GetDebugInfo());
         }
 
+        [Fact]
+        public void Builds_NetCore31App_UsingNetCore31_DotNetSdkVersion_CustomError()
+        {
+            // Arrange
+            var appName = "NetCoreApp31.MvcApp";
+            var volume = CreateSampleAppVolume(appName);
+            var appDir = volume.ContainerDir;
+            var appOutputDir = "/tmp/NetCoreApp31MvcApp-output";
+            var script = new ShellScriptBuilder()
+                .AddCommand($"echo RandomText >> {appDir}/Program.cs") // triggers a failure
+                .AddBuildCommand($"{appDir} -o {appOutputDir} --platform dotnet --platform-version 3.1.8")
+                .ToString();
+            // Regex will match:
+            // "yyyy-mm-dd hh:mm:ss"|ERROR|Micro
+            Regex regex = new Regex(@"""[0-9]{4}-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1]) (0[0-9]|1[0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9])""\|ERROR\|.*");
```

Comment:
@william-msft : Just wanted to config, we print errors in the following format right ?
 "2022-03-29 05:49:25"|ERROR| Message | Exit code: 1 | Recommendation | More information: Link

Reply:
Confirmed with Vijay offline: this will look similar to `"2022-03-29 05:49:25"|ERROR| {message}| {exitCode}| {recommendation}| {doc}`